services/server/src/server/services/storageServices/RepositoryV1Service.ts

```diff
@@ -112,6 +163,7 @@ export class RepositoryV1Service implements IStorageService {
     for (const file in files) {
       const loadedFile = fs.readFileSync(files[file].path);
```

Comment:
Let's remove readSync here. This can heavily degrade performance. Can you also look for other places if there's sync I/O. IMO it's important

Reply:
Should we do this in this PR?

Reply:
The ones in services, yes. Others (tests, lib-sourcify etc.) can be left out. I see there are not too many of them in services.